app/scripts/views/mixins/resume-token-mixin.js

```diff
@@ -40,10 +40,11 @@ define(function (require, exports, module) {
      * Get a stringified ResumeToken that can be passed along in an email
      *
      * @method getStringifiedResumeToken
+     * @param {Object} account
      * @returns {String}
      */
-    getStringifiedResumeToken () {
-      return this.getResumeToken().stringify();
+    getStringifiedResumeToken (account) {
+      return this.getResumeToken(account).stringify();
```

Comment:
Any chance of account being undefined here? Should we have any guards against that?